packages/extensionmanager/src/widget.tsx

```diff
@@ -740,7 +740,6 @@ export class ExtensionsPanel extends SidePanel {
       (this.content as AccordionPanel).collapse(0);
       (this.content as AccordionPanel).expand(1);
       (this.content as AccordionPanel).expand(2);
-      (this.content.layout as AccordionLayout).setRelativeSizes([0, 1, 1]);
```

Comment:
Could we keep this line but update the `_wasDisclaimed` status so that none of these will execute? Something like:

```ts
     (this.content.layout as AccordionLayout).setRelativeSizes([0, 1, 1]);
  }
  this._wasDisclaimed = this.model.isDisclaimed;
```

Also, I would rename this method to `_onStateChanged` because in fact it is run on every state update which is likely why this bug has arisen.

Reply:
Yes, your approach seems to be in line with the original design idea, but I don't understand what the function of line 743

Reply:
Yes, your writing is correct to avoid unnecessary code execution, but I still don’t understand the function of line 743. It seems that the relative size setting only needs to be done during initialization